modules/ROOT/pages/test-mule-applications.adoc

```diff
@@ -0,0 +1,63 @@
+= Test Mule Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Building a comprehensive suite of automated tests for a Mule Application enables you to detect any regression or incompatible change in your application during tests in a local environment.
```

Comment:
This is more editorial, but in this first sentence I would a mention that you will cover unit, integration, writing testable code and its tips information, also about MUnit and Profiling Mule since these two jump directly after it. Would these two product/features be considered automated tests? You can then add something like...Build automated test using MUnit or Profiling Mule...something like that.